package.json

```diff
@@ -32,6 +32,8 @@
     "webpack-dev-server": "^3.1.14"
   },
   "dependencies": {
-    "jquery": "^3.4.1"
+    "flatpickr": "^4.6.3",
```

Comment:
i think we can remove flatpickr from the dependencies. Ill see how we can do that.